app/views/bundle_contexts/show.html.erb

```diff
@@ -1,26 +1,5 @@
 <div class="container mt-5">
-  <dl class="dl row">
-    <dt class="col-sm-3">Project Name</dt>
-    <dd class="col-sm-9"><%= @bundle_context.project_name %></dd>
-    <dt class="col-sm-3">User</dt>
-    <dd class="col-sm-9"><%= @bundle_context.user.email %></dd>
-    <dt class="col-sm-3">Content Structure</dt>
-    <dd class="col-sm-9"><%= @bundle_context.content_structure %></dd>
-    <dt class="col-sm-3">Bundle Directory</dt>
-    <dd class="col-sm-9"><%= @bundle_context.bundle_dir %></dd>
-    <dt class="col-sm-3">Staging Style Symlink?</dt>
-    <dd class="col-sm-9">
-      <% if @bundle_context.staging_style_symlink %>
-        <i class="fas fa-check"></i>
-      <% else %>
-        <i class="far fa-times-circle"></i>
-      <% end %>
-    </dd>
-    <dt class="col-sm-3">Content Metadata Creation</dt>
-    <dd class="col-sm-9"><%= @bundle_context.content_metadata_creation %></dd>
-    <dt class="col-sm-3">Created At</dt>
-    <dd class="col-sm-9"><%= @bundle_context.created_at %></dd>
-  </dl>
+  <%= render @bundle_context %>
```

Comment:
Note the cleanliness of following the rails convention here and in the other template).  The `ActiveModel` object knows where its own partial lives.